server/server.go

```diff
@@ -146,6 +146,7 @@ func NewServer(ctx *Context, stopper *stop.Stopper) (*Server, error) {
 	retryOpts.Closer = stopper.ShouldDrain()
```

Comment:
Ah, right. So I guess what I should do instead of introducing the new field is make all the tests set RPCRetryOptions appropriately.